arangod/Cluster/ClusterFeature.cpp

```diff
@@ -104,7 +105,8 @@ void ClusterFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
 
   options->addOption("--cluster.agency-prefix", "agency prefix",
                      new StringParameter(&_agencyPrefix),
-                     arangodb::options::makeFlags(arangodb::options::Flags::Hidden));
+                     makeFlags(arangodb::options::Flags::Hidden,
+                               arangodb::options::Flags::Obsolete));
```

Comment:
We need to document that the option becomes obsolete in 3.7, in both CHANGELOG and the upgrading-changes document for 3.7.
If the option was obsolete even before 3.7, we should document this as well.